### PR TITLE
PLG-33 Sumo Logic Sample Data (Astronomy) app doc

### PR DESCRIPTION
**Pull request by ishaanahuja29**
## Purpose of this pull request

This pull request is to create apps doc for Sumo Logic Sample Data (Astronomy) app.
NOTE: Its not required for all customers, so I've not added it to sidebar/ product list. It'll be accessed via link directly.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)
[PLG-33](https://sumologic.atlassian.net/browse/PLG-33)

### CODE REVIEW

File: docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Sumo Logic Sample Data (Astronomy) - Application Health
+
+The **Sumo Logic Sample Data (Astronomy) - Application Health Dashboard** provides a comprehensive view of the application's performance and operational metrics. It includes key indicators such as CPU and memory usage, transaction times for checkouts and payments, error rates, and service latency. Additionally, it displays deployment logs, user journeys, and error logs, allowing users to quickly identify issues and monitor the overall health of the application.
+
+<img src={useBaseUrl('img/integrations/sumo-apps/Sumo-Logic-Sample-Data-Astronomy-Application-Health-Dashboard.png')} alt="test" />
```
Comment (jpipkin1) [inline docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md:40]:
```suggestion
<img src={useBaseUrl('img/integrations/sumo-apps/Sumo-Logic-Sample-Data-Astronomy-Application-Health-Dashboard.png')} style={{border: '1px solid gray'}} alt="test" />
```

**Review by jpipkin1 — APPROVED** [review]
This article looks very good, @ishaanahuja29. I had only a couple of minor suggestions about adding borders to images.

File: sidebars.ts [hunk 2]
```diff
@@ -2401,6 +2401,7 @@ integrations: [
           'integrations/sumo-apps/infrequent-data-tier',
           'integrations/sumo-apps/log-analysis-quickstart',
           'integrations/sumo-apps/security-analytics',
+          // 'integrations/sumo-apps/sumo-logic-sample-data-astronomy',
```
Comment (kimsauce) [inline sidebars.ts:2404]: @ishaanahuja29 , to confirm, this app is GA, not beta? If GA, you'd need to apply this change.

```suggestion
          'integrations/sumo-apps/sumo-logic-sample-data-astronomy',
```

File: cid-redirects.json [hunk 3]
```diff
@@ -2375,6 +2375,7 @@
   "/cid/5421": "/docs/search/search-query-language/search-operators/fillmissing",
   "/cid/5422": "/docs/search/time-compare",
   "/cid/12356": "/docs/integrations/sumo-apps/log-analysis-quickstart",
+  "/cid/12357": "/docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy",
```
Comment (kimsauce) [inline cid-redirects.json:2378]: For Sumo Logic apps URLs, we omit the "sumo-logic-" prefix to keep it clean. See [examples](https://help.sumologic.com/docs/integrations/sumo-apps).

```suggestion
  "/cid/12357": "/docs/integrations/sumo-apps/sample-data-astronomy",
```

File: docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+id: sumo-logic-sample-data-astronomy
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md:2]: For Sumo Logic apps URLs, we omit the "sumo-logic-" prefix to keep it clean. See [examples](https://help.sumologic.com/docs/integrations/sumo-apps).

```suggestion
id: sample-data-astronomy
```

File: docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+The **Sumo Logic Sample Data (Astronomy) - Application Health Dashboard** provides a comprehensive view of the application's performance and operational metrics. It includes key indicators such as CPU and memory usage, transaction times for checkouts and payments, error rates, and service latency. Additionally, it displays deployment logs, user journeys, and error logs, allowing users to quickly identify issues and monitor the overall health of the application.
+
+<img src={useBaseUrl('img/integrations/sumo-apps/Sumo-Logic-Sample-Data-Astronomy-Application-Health-Dashboard.png')} alt="test" />
+
+### Sumo Logic Sample Data (Astronomy) - Business KPIs
+
+The **Sumo Logic Sample Data (Astronomy) - Business KPIs Dashboard** focuses on critical business performance metrics. It tracks customer satisfaction, support tickets, revenue trends, and promotional offers redeemed. This dashboard also monitors customer churn rates and cart abandonment rates, providing insights into user engagement and revenue distribution globally. These metrics help businesses make informed decisions to enhance customer experiences and drive growth.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md:44]:
```suggestion
The **Sumo Logic Sample Data (Astronomy) - Business KPIs** dashboard focuses on critical business performance metrics. It tracks customer satisfaction, support tickets, revenue trends, and promotional offers redeemed. This dashboard also monitors customer churn rates and cart abandonment rates, providing insights into user engagement and revenue distribution globally. These metrics help businesses make informed decisions to enhance customer experiences and drive growth.
```

File: docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+## Viewing Sumo Logic Sample Data (Astronomy) dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Sumo Logic Sample Data (Astronomy) - Application Health
+
+The **Sumo Logic Sample Data (Astronomy) - Application Health Dashboard** provides a comprehensive view of the application's performance and operational metrics. It includes key indicators such as CPU and memory usage, transaction times for checkouts and payments, error rates, and service latency. Additionally, it displays deployment logs, user journeys, and error logs, allowing users to quickly identify issues and monitor the overall health of the application.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy.md:38]:
```suggestion
The **Sumo Logic Sample Data (Astronomy) - Application Health** dashboard provides a comprehensive view of the application's performance and operational metrics. It includes key indicators such as CPU and memory usage, transaction times for checkouts and payments, error rates, and service latency. Additionally, it displays deployment logs, user journeys, and error logs, allowing users to quickly identify issues and monitor the overall health of the application.
```

File: docs/integrations/sumo-apps/sample-data-astronomy.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+---
+id: sumo-logic-sample-data-astronomy
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sample-data-astronomy.md:2]: For Sumo Logic apps URLs, we omit the "sumo-logic-" prefix to keep them clean. See [examples](https://help.sumologic.com/docs/integrations/sumo-apps).

```suggestion
id: sample-data-astronomy
```

File: docs/integrations/sumo-apps/sample-data-astronomy.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+<img src={useBaseUrl('img/integrations/sumo-apps/Sumo-Logic-Sample-Data-Astronomy-Application-Health-Dashboard.png')} alt="test" />
+
+### Sumo Logic Sample Data (Astronomy) - Business KPIs
+
+The **Sumo Logic Sample Data (Astronomy) - Business KPIs Dashboard** focuses on critical business performance metrics. It tracks customer satisfaction, support tickets, revenue trends, and promotional offers redeemed. This dashboard also monitors customer churn rates and cart abandonment rates, providing insights into user engagement and revenue distribution globally. These metrics help businesses make informed decisions to enhance customer experiences and drive growth.
+
+<img src={useBaseUrl('img/integrations/sumo-apps/Sumo-Logic-Sample-Data-Astronomy-Business-KPIs-Dashboard.png')} alt="test" />
+
+## Upgrading the Sumo Logic Sample Data (Astronomy) app (Optional)
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sample-data-astronomy.md:48]:
```suggestion
## Upgrading the Sumo Logic Sample Data (Astronomy) app (optional)
```

File: docs/integrations/sumo-apps/sample-data-astronomy.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+import AppUpdate from '../../reuse/apps/app-update.md';
+
+<AppUpdate/>
+
+## Uninstalling the Sumo Logic Sample Data (Astronomy) app (Optional)
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/sample-data-astronomy.md:54]:
```suggestion
## Uninstalling the Sumo Logic Sample Data (Astronomy) app (optional)
```

**Pull request merged**